easysnmp/interface.c

```diff
@@ -444,7 +444,7 @@ static int __snprint_value(char *buf, size_t buf_len,
             case ASN_TIMETICKS:
             case ASN_UINTEGER:
                 snprintf(buf, buf_len, "%lu",
-                         (unsigned long) *var->val.integer);
+                         (ulong) *var->val.integer);
```

Comment:
This is unnecessary churn. Could you please change the cast back to `unsigned long` here?

Reply:
done, thanks